src/python/pants/bsp/protocol.py

```diff
@@ -155,8 +155,7 @@ def _handle_inbound_message(self, *, method_name: str, params: Any):
         workspace = Workspace(self._scheduler_session)
         params = Params(request, workspace)
         execution_request = self._scheduler_session.execution_request(
-            products=[method_mapping.response_type],
-            subjects=[params],
+            requests=[(method_mapping.response_type, params)],
         )
         returns, throws = self._scheduler_session.execute(execution_request)
```

Comment:
The return type for `.execute` appears to have been changed to return `list[any]` and to raise any of the `throws`. If so, this line will need modification, correct?

Reply:
Thanks: caught by CI too luckily.